packages/orbit-components/src/ErrorFormTooltip/Tooltip/types.d.ts

```diff
@@ -9,7 +9,6 @@ export interface Props extends Common.Globals {
   readonly helpClosable: boolean;
   readonly isHelp?: boolean;
   readonly referenceElement?: { current: HTMLElement | null };
-  readonly inputSize?: Common.InputSize;
```

Comment:
Commit message mentions `size type` prop but it's actually called `inputSize prop`. The BC message should also mention "prop" instead of "type".